libs/genai/langchain_google_genai/_function_utils.py

```diff
@@ -277,15 +355,18 @@ def _get_type_from_schema(schema: Dict[str, Any]) -> int:
             pass
     elif "type" in schema:
         stype = str(schema["type"])
-        if stype in TYPE_ENUM:
-            return TYPE_ENUM[stype]
-        else:
-            pass
+        return _get_type_from_str(stype)
```

Comment:
can we maybe just have a one-liner here instead creating a separate function?
`TYPE_ENUM.get(stype, "str")`

Reply:
@lkuligin 
I fixed as one-liner.